src/crimson/osd/ops_executer.cc

```diff
@@ -1089,11 +1089,15 @@ ObjectContextRef OpsExecuter::prepare_clone(
   clone_obs.oi.copy_user_bits(initial_obs.oi);
   clone_obs.oi.clear_flag(object_info_t::FLAG_WHITEOUT);
 
-  auto [clone_obc, existed] = pg->obc_registry.get_cached_obc(std::move(coid));
+  auto [clone_obc, existed] = pg->obc_registry.get_cached_obc(coid);
+  std::lock_guard<std::mutex> lock(clone_mutex);
```

Comment:
std::mutex can't be used in Crimson. std::mutex is used for traditional multi-threading. In Crimson the Seastar framework uses a single thread (asynchronous thread). Using std::mutex will block the entire Seastar reactor thread. Seastar provide seastar::mutex to handle locking - however, iiuc, the obc locks which already exist should handle this case (see previous comment)